### PR TITLE
change management/index.js to functional behavior

### PR DESCRIPTION
**Pull request by dhiraj0911**
Signed-off-by: Dhiraj Bari <djirajbari123@gmail.com>

#  Convert management/index.js to Functional Component

**Description:**

This PR converts the `management/index.js` file to a functional component for improved maintainability and readability. It addresses issue #8676.

**Changes:**

- Converted the existing class-based component to a functional component.
- Utilized React hooks to manage state and side effects.
- Updated the document title in a useEffect hook.

**Testing:**

I have tested the changes locally to ensure that the component functions correctly and that there are no console errors or warnings.

**Signed Commits:**

- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/pages/management/index.js [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
-      <NoSsr>
-        <Head>
-          <title>Management | Meshery </title>
-        </Head>
-        <MesheryPlayComponent />
-      </NoSsr>
-    );
-  }
+
+function Manage(props) {
```
Comment (Rajdip019) [inline ui/pages/management/index.js:10]: Is Props are getting used? 

If not I can we can safely remove that.

**Comment by Rajdip019** [comment]
The ES-Lint check has to pass.  @dhiraj0911

File: ui/pages/management/index.js (hunk 1 above)
Comment (dhiraj0911) [reply ui/pages/management/index.js:10]: Sure, thanks

**Comment by dhiraj0911** [comment]
Will fix it

File: ui/pages/management/index.js [hunk 2]
```diff
@@ -1,27 +1,31 @@
...
+  useEffect(() => {
+    document.title = "Management | Meshery";
```
Comment (Ckbhatia) [inline ui/pages/management/index.js:12]: I think this should be managed at the routing level for simplicity

**Comment by leecalcote** [comment]
Thanks, @dhiraj0911!

**Pull request closed without merging**